tarantool_test.go

```diff
@@ -693,6 +694,514 @@ func TestClient(t *testing.T) {
 	}
 }
 
+const (
+	createTableQuery    = "CREATE TABLE SQL_SPACE (id INTEGER PRIMARY KEY AUTOINCREMENT, name STRING COLLATE \"unicode\" DEFAULT NULL);"
+	insertQuery         = "INSERT INTO SQL_SPACE VALUES (?, ?);"
+	selectNamedQuery    = "SELECT id, name FROM SQL_SPACE WHERE id=:id AND name=:name;"
+	selectPosQuery      = "SELECT id, name FROM SQL_SPACE WHERE id=? AND name=?;"
+	updateQuery         = "UPDATE SQL_SPACE SET name=? WHERE id=?;"
+	enableFullMetaData  = "SET SESSION \"sql_full_metadata\" = true;"
+	selectSpanDifQuery  = "SELECT id*2, name, id FROM SQL_SPACE WHERE name=?;"
+	alterQueryIncrement = "ALTER TABLE SQL_SPACE RENAME TO SQL_SPACE2;"
+	insertIncrQuery     = "INSERT INTO SQL_SPACE2 VALUES (?, ?);"
+	deleteQuery         = "DELETE FROM SQL_SPACE2 WHERE name=?;"
+	dropQuery           = "DROP TABLE SQL_SPACE2;"
+	dropQuery2          = "DROP TABLE SQL_SPACE;"
+	disableFullMetaData = "SET SESSION \"sql_full_metadata\" = false;"
+)
+
+func TestSQL(t *testing.T) {
+	// Check for skip SQL tests if tarantool version < 2.0.0
+	isLess, err := test_helpers.IsTarantoolVersionLess(2, 0, 0)
+	if err != nil {
+		t.Errorf("Could not check the Tarantool version")
+		return
+	}
+	if isLess {
+		return
+	}
+
+	queries := []string{
+		createTableQuery,
+		insertQuery,
+		selectNamedQuery,
+		selectPosQuery,
+		updateQuery,
+		enableFullMetaData,
+		selectSpanDifQuery,
+		alterQueryIncrement,
+		insertIncrQuery,
+		deleteQuery,
+		dropQuery,
+		disableFullMetaData,
```

Comment:
Nit (feel free to ignore): names looks inconsistent:

- `createTableQuery` (Query at the end of name)
- `enableFullMetaData` (Query is not a part of name)
- `alterQueryIncrement` (Query in the middle of name)

Reply:
Fixed.

Reply:
Seems the change was to replace `alterQueryIncrement` to `alterTableQuery`, right?